vip-helpers/class-user-cleanup.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Automattic\VIP\Helpers;
+
+// If the constant is set, don't run the cleanup
+if ( defined( 'VIP_SKIP_USER_CLEANUP' ) && true === VIP_SKIP_USER_CLEANUP ) {
+	add_filter( 'vip_do_user_cleanup', '__return_false' );
+}
+
+class User_Cleanup {
+	public static function parse_emails_string( $emails_string ) {
+		$emails = [];
+
+		$emails_string = trim( $emails_string );
+
+		if ( false !== strpos( $emails_string, ',' ) ) {
+			$emails_raw = explode( ',', $emails_string );
+		} else {
+			$emails_raw = [ $emails_string ];
+		}
+
+		foreach ( $emails_raw as $email_raw ) {
+			$email_raw = trim( $email_raw );
+
+			$email_filtered = filter_var( $email_raw, FILTER_VALIDATE_EMAIL );
+
+			if ( empty( $email_filtered ) ) {
```

Comment:
Small nitpick, but since the filter_var's return signature is `bool|string` let's check against that.
```suggestion
			if ( false === $email_filtered  ) {
```
Alternatively can flip the logic and make a truthy comparison and push into the array there, two less LoC :)

Reply:
`empty` is probably better though in the off-chance that filter_var somehow returns `''`?

Reply:
But the "truthy" check might be better.

Reply:
I don't believe so. An empty string would result in `false` because it's not a valid email.